scheduler/kubernetes/install.md

```diff
@@ -4,14 +4,7 @@ title: "Deploy Portworx on Kubernetes"
 keywords: portworx, container, Kubernetes, storage, Docker, k8s, flexvol, pv, persistent disk
 sidebar: home_sidebar
 redirect_from:
-  - /gce-k8s-pwx.html
```

Comment:
we should keep these in place if these links ever existed.  Otherwise we might get links that 404 to our docs.  Alternatively, to you move those redirects to another page.